src/Scheduling/MassTransit.QuartzIntegration/MassTransitJobFactory.cs

```diff
@@ -90,7 +97,7 @@ public void ReturnJob(IJob job)
         {
         }
 
-        void SetObjectProperties(T job, JobDataMap jobData)
+        static void SetObjectProperties(T job, JobDataMap jobData)

```

Comment:
Here I was hoping to create a Quartz implementation that is good enough out of the box, doesn't affect this PR though